pom.xml

```diff
@@ -76,6 +76,12 @@
         <!-- API checker -->
         <revapi-java.version>0.18.0</revapi-java.version>
         <revapi.plugin.version>0.10.5</revapi.plugin.version>
+
+        <!-- Sonar properties -->
```

Comment:
You should specify here the `sonar.projectKey` property:
`<sonar.projectKey>apache_incubator-tamaya</sonar.projectKey>`